LocalWiki.php

```diff
@@ -191,6 +191,11 @@ function onBeforePageDisplay( OutputPage $outputPage ) {
 			'url' => 'https://gpcommons.miraheze.org/w/api.php'
 		];
 
+		break;
+	case 'hommwiki':
+		$wgEnabledAudioTranscodeSet = [];
+		$wgEnabledTranscodeSet = [];
```

Comment:
This doesn't seem to be necessary? It seems just $wgEnabledAudioTranscodeSet along side $wgEnableTranscode (which is available in ManageWiki)

Reply:
I did it just in case, but I suppose I can remove it.